# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1295085462 18000
#      Sat Jan 15 04:57:42 2011 -0500
# Node ID 2c5219471c861829f53a8eae7a34bf859db982e8
# Parent  695141f1c05cf1b240592bdd18e7a1503bb2a539
Added tag ss-3-3-55 for changeset 695141f1c05c

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -44,8 +44,9 @@ 3c3cbe8f18e03c73cc34494c8037d8a45a30bf7d
 5e276a0b999747b72af34e72eb63f64640da6399 ss-3-1-55
 a1d20052517a950033896f5a876de8f3ee520138 release-3-2-0
 74be4b7273e43952e1afcd9ad979d35667c7f82e ss-3-3-50
 79a56d0a6a0d693ebf461c456951b20dfbcde140 ss-3-3-51
 229675bb7647f7e7e499bcb5acacc8eabe7f3697 ss-3-3-52
 6c69a7c39039bccd5a2fe481fcc7f896c82b160b ss-3-3-53
 3cbc0d77db48aec32bcb202d09a036d2cb9cc3b9 ss-3-3-53
 bd2643f0ce57d070963bedd48857405f6924aa85 ss-3-3-54
+695141f1c05cf1b240592bdd18e7a1503bb2a539 ss-3-3-55
